chandra_aca/tests/test_maude_decom.py

```diff
@@ -1185,3 +1185,29 @@ def test_filter_vcdu_jumps():
     for case in filter_vcdu_test_cases:
         idx = maude_decom.filter_vcdu_jumps(case["inp"])
         assert np.all(idx == case["exp"])
+
+
+def test_end_integ_time():
```

Comment:
This test does check for the correct relationship between `TIME` and `END_INTEG_TIME` for L0, but does not show that those times match actual L0 data from the CXC archive. That it is the real requirement, namely that the telemetry data, times, and the image values all match.

I see references to similar testing with `test_data` taken from a pickle, but I don't see any docs on how that was generated (maybe I just did find them?). But since this change didn't break any of those tests then it must be that `END_INTEG_TIME` was not previously covered.

Reply:
Yes, I think that comparison needs checking now that END_INTEG_TIME changed.

Reply:
I added a test to compare the times from `chandra_aca.aude_decom.get_aca_images` and `mica.archive.aca_l0.get_images`.

I still have not checked why tests did not fail when END_INTEG_TIME changed.